tests/virt_autotest/host_upgrade_base.pm

```diff
@@ -8,8 +8,8 @@
 # without any warranty.
 #
 package host_upgrade_base;
-# G-Summary: virt_autotest: the initial version of virtualization automation test in openqa, with kvm support fully, xen support not done yet
-# G-Maintainer: alice <xlai@suse.com>
+# Summary: virt_autotest: virtualization automation test in openqa, support kvm and xen.
```

Comment:
please find a unique summary line for all of the modules. It does not make sense if they are all identical.